hazelcast/src/test/java/com/hazelcast/test/starter/HazelcastVersionLocator.java

```diff
@@ -56,14 +56,14 @@
     }
 
     public static File[] locateVersion(String version, File target, boolean enterprise) {
-        File[] files = new File[enterprise ? 6 : 3];
+        File[] files = new File[enterprise ? 4 : 2];
         files[0] = locateMember(version, target, false);
         files[1] = locateMemberTests(version, target, false);
-        files[2] = locateClient(version, target, false);
+        //files[2] = locateClient(version, target, false);
```

Comment:
Maybe only exclude the client artifact when `version` is 4.x? Otherwise the `hazelcast-client` artifact will be missing from 3.12-3.11 compat tests, should we want to execute them.

Reply:
Sure, I think I had a similar idea, will do it.

Reply:
Fixed.